feature-libs/cart/base/core/facade/active-cart.service.ts

```diff
@@ -64,9 +66,14 @@ export class ActiveCartService implements ActiveCartFacade, OnDestroy {
     switchMap((cartId) => this.multiCartFacade.getCartEntity(cartId))
   );
 
+  // Flag to prevent cart loading when logged in with code flow
+  // Instead of loading cart will run loadOrMerge method
+  protected shouldLoadCartOnCodeFlow = true;
+
   constructor(
     protected multiCartFacade: MultiCartFacade,
-    protected userIdService: UserIdService
+    protected userIdService: UserIdService,
+    protected winRef: WindowRef
```

Comment:
Can we mark this parameter as an optional, with a comment that might be required from the next major version? I am just thinking how to avoid a breaking change here.

Reply:
Done